utils.py

```diff
@@ -0,0 +1,137 @@
+import ast
+import os
+import random
+import hashlib
+import hmac
+import pytz
+
+from datetime import date
+from datetime import datetime
+from datetime import timedelta
+from pytz import timezone
+
+slack_signing_secret = os.environ["SLACK_SIGNING_SECRET"]
+team_list_var = os.environ["TEAM_LIST"]
+
+
+def create_markdown_list(elements):
+    if not elements:
+        return ""
+
+    # Slack doesn't support actual markdown lists so I'm mocking a list
+    # with a "-" char before each element and a new line after
+    # https://api.slack.com/reference/surfaces/formatting#block-formatting
+    markdown_list = ""
+
+    for element in elements:
+        markdown_list += f"- {element}\n "
+
+    return markdown_list
+
+
+def generate_demo_order_blocks(header_text="Here's a list of teams in random order:"):
+    team_list = generate_randomized_list(team_list_var)
+    return [
+        {
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": header_text,
+            },
+        },
+        {
+            "type": "section",
+            "text": {"type": "mrkdwn", "text": create_markdown_list(team_list)},
+        },
+    ]
+
+
+def generate_randomized_list(team_list):
+    # Add check for team list after its stored in db
+    teams = ast.literal_eval(team_list)
+    return random.sample(teams, len(teams))
+
+
+def generate_valid_dates():
```

Comment:
Is this just a helper function you used to populate the list below? Don't think it hurts to have it but it could theoretically be removed if we're just hard-coding a list of acceptable dates that ends at some point 😅.